geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SelectionUtils.js

```diff
@@ -0,0 +1,40 @@
+import { hexToRGBNormalized } from "../../threeDEngine/util";
+
+const SELECTION_COLOR = { r: 0.8, g: 0.8, b: 0, a: 1 };
+
+
+export function dataMapping (data){
```

Comment:
would prefer `createCanvasData` or `mapToCanvasData`, dataMapping is not an "action expression"